ivadomed/config/config_spineGeHemis.json

```diff
@@ -3,6 +3,7 @@
     "gpu": 0,
     "log_directory": "HeMIS",
     "debugging": false,
+    "object_detection_path": null,
```

Comment:
What about having directly in the config file / loader_params a dict `object_detection_params` with `path`. Then in main, add to this dict "gpu" and "log_directory" etc.?

Reply:
Fixed :)